kitt_bar_app/current_batch.rb

```diff
@@ -86,6 +95,11 @@ def user_ticket
     @api_data.dig('tickets')&.find { |ticket| ticket.dig('is_mine') }
   end
 
+  def assigned_ticket(ticket)
+    return if ticket.dig('assigned', 'id').nil?
```

Comment:
Style: leave an empty line after guard close #rubocopped 🤓

Reply:
Will correct that !